eigenda/config.go

```diff
@@ -1,41 +1,37 @@
 package eigenda
 
 import (
-	"errors"
 	"runtime"
 	"time"
 
+	"github.com/Layr-Labs/eigenda/api/clients"
+	"github.com/Layr-Labs/eigenda/api/clients/codecs"
 	"github.com/Layr-Labs/eigenda/encoding/kzg"
 	opservice "github.com/ethereum-optimism/optimism/op-service"
 	"github.com/urfave/cli/v2"
 )
 
 const (
-	RPCFlagName                      = "eigenda-rpc"
-	StatusQueryRetryIntervalFlagName = "eigenda-status-query-retry-interval"
-	StatusQueryTimeoutFlagName       = "eigenda-status-query-timeout"
-	UseTlsFlagName                   = "eigenda-use-tls"
+	RPCFlagName                          = "eigenda-rpc"
+	StatusQueryRetryIntervalFlagName     = "eigenda-status-query-retry-interval"
+	StatusQueryTimeoutFlagName           = "eigenda-status-query-timeout"
+	DisableTlsFlagName                   = "eigenda-disable-tls"
+	ResponseTimeoutFlagName              = "eigenda-response-timeout"
+	CustomQuorumIDsFlagName              = "eigenda-custom-quorum-ids"
+	SignerPrivateKeyHexFlagName          = "eigenda-signer-private-key-hex"
+	PutBlobEncodingVersionFlagName       = "eigenda-put-blob-encoding-version"
+	DisablePointVerificationModeFlagName = "eigenda-disable-point-verification-mode"
```

Comment:
Let's update `README.md` to reflect any of these new values which are required for operating the service (e.g, SignerPrivateKeyHexFlagName)